app/controllers/application_controller/miq_request_methods.rb

```diff
@@ -583,8 +583,7 @@ def prov_request_cancel_submit_response
         else
           # had to get id from breadcrumbs url,
           # because there is no params[:id] when cancel is pressed on copy Request screen.
-          url = @breadcrumbs.last[:url].split('/')
-          page.redirect_to :controller => url[1], :action => url[2], :id => url[3]
+          page.redirect_to @breadcrumbs.last[:url]
```

Comment:
> @AparnaKarve does the 2-line comment above the code still make sense?

@chessbyte, You're right, the comment does not make sense and should be removed. Thanks.